ComSemApp/templates/ComSemApp/teacher/request_list.html

```diff
@@ -0,0 +1,55 @@
+{% if requests %}
+<table class="table table-hover tablesorter">
+    <thead>
+        <tr>
+            <th>Date</th>
+            <th>Student</th>
+            <th>Problem Expression</th>
+            <th>Reviewed</th>
+            <th></th>
```

Comment:
why the empty `<th>`?

Reply:
Formatting. that column holds the "edit" button